experimental/streaming/exporter/reader/format/format.go

```diff
@@ -133,6 +133,32 @@ func AppendEvent(buf *strings.Builder, data reader.Event) {
 	buf.WriteString(" ]\n")
 }
 
+func formatMetricUpdate(buf *strings.Builder, m metric.Measurement) {
+	buf.WriteString(m.Instrument.Kind.String())
+	buf.WriteString(" ")
+	buf.WriteString(m.Instrument.Name)
+	buf.WriteString("=")
+	buf.WriteString(fmt.Sprint(m.Value))
+}
+
+func formatMetricLabels(buf *strings.Builder, l tag.Map) {
+	buf.WriteString(" {")
+	i := 0
+	// Note: This if-conditional can be removed after Map is not
+	// an interface.  Will `m.Tags()` should work for a nil map?
```

Comment:
Does this comment still apply? There is no `m` in scope here.

Reply:
Fixed.